AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/GetUnallocatedHearingsTests.cs

```diff
@@ -29,8 +29,8 @@ public void Setup()
     public async Task Should_get_unallocated_hearings()
     {
         // Arrange
-        _mocker.Mock<IBookingsApiClient>().Setup(bs => bs.GetUnallocatedHearingsAsync())
-            .ReturnsAsync(new List<HearingDetailsResponse>());
+        _mocker.Mock<IBookingsApiClient>().Setup(client => client.GetUnallocatedHearingsAsync())
```

Comment:
could you add an mocked to to return when you call a client rather than an empty list else it's very similar to the test below

Reply:
will update it